mas/src/mas.js

```diff
@@ -10,14 +8,40 @@ const features = searchParams.get('features');
 const envName = isStage ? 'stage' : 'prod';
 const commerceEnv = isStage ? 'STAGE' : 'PROD';
 
+function loadScript(url) {
+    return new Promise((resolve, reject) => {
+        const script = document.createElement('script');
+        script.src = url;
+        script.onload = () => resolve();
+        script.onerror = () =>
+            reject(new Error(`Failed to load script: ${url}`));
+        document.head.appendChild(script);
+    });
+}
+
 const config = () => ({
     env: { name: envName },
     commerce: { 'commerce.env': commerceEnv },
     locale: { prefix: locale },
 });
 
-init(config);
+(async () => {
+    try {
+        await loadScript(
```

Comment:
can this URL be put in a config file and based on we are running on local or hosted, we can pick the intended URL

Reply:
actually it should either be www.stage.adobe.com or www.adobe.com.
We can change it later as well once we decide how to handle environments on MAS platform.

Reply:
fixed